mountainduck/preferences.md

```diff
@@ -43,15 +43,21 @@ Enable to prevent conflicting edits when accessing documents from a shared envir
 
 ### Mount Location 
 
-````{admonition} macOS only
+```{admonition} 
 :class: tip
 
+Only available on macOS using the standalone version.
```

Comment:
We have never referred to `standalone` for the regular version of Mountain Duck distributed outside of the Mac App Store. I would instead add a note when a setting does not apply to the Mac App Store version and is therefore disabled in Preferences.